haproxy.py

```diff
@@ -8,7 +8,10 @@
 #
 # Modified by "Warren Turkal" <wt@signalfuse.com>, "Volodymyr Zhabiuk" <vzhabiuk@signalfx.com>
 
-import cStringIO as StringIO
+try:
+    import cStringIO as StringIO
+except ImportError:
+    from io import StringIO
```

Comment:
```suggestion
import io
```

then use `io.BytesIO` later in the file